### PR TITLE
1st App iteration: Connection of MM wallet & generate signed siwe message

### PR DESCRIPTION
**Pull request by filipegoncalvesBP**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Domain options
+# DEV: 
+# core: https://dev.getunblock.com 
+# fiat-connect: https://dev.getunblock.com/fiat-connect/
+# STAGING:
```
Comment (Przemek-Win) [inline .env.example]: Call it Sandbox

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Domain options
+# DEV: 
+# core: https://dev.getunblock.com 
+# fiat-connect: https://dev.getunblock.com/fiat-connect/
+# STAGING: 
+# core: https://sandbox.getunblock.com
+# fiat-connect: https://sandbox.getunblock.com/fiat-connect
+# Prod:
```
Comment (Przemek-Win) [inline .env.example]: Capital letter as well

File: .env.example (hunk 1 above)
Comment (filipegoncalvesBP) [reply .env.example]: Got it 👍

File: .env.example (hunk 2 above)
Comment (filipegoncalvesBP) [reply .env.example]: Yeah not sure why I did not put it capital as well 👍

**Review by Przemek-Win — DISMISSED** [review]

**Review by Przemek-Win — APPROVED** [review]

**Pull request merged**